recipes/libtiff/all/test_package/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package LANGUAGES C)
+cmake_minimum_required(VERSION 3.15)
+project(test_package C CXX)
 
 find_package(TIFF REQUIRED)
 
-add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} PRIVATE TIFF::TIFF)
+add_executable(${PROJECT_NAME}_c test_package.c)
+target_link_libraries(${PROJECT_NAME}_c PRIVATE TIFF::TIFF)
+set_target_properties(${PROJECT_NAME}_c PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
```suggestion
```

It shouldn't be needed, otherwise there is an issue.

Reply:
Oh, thanks! That's a leftover from debugging. Did not intend to leave it in.